### ISSUE TITLE
Support for disabling difficulties

### ISSUE THREAD

**Issue by Toasty-Engineer** [William758/ContentDisabler#5]
I'd like to be able to disable/hide difficulties from the menu; I prefer using Inferno Mode over Typhoon and I'd like to be able to hide the latter instead of having a redundant mode cluttering the menu.

**Comment by William758** [comment]
Yeah I'll go ahead and add support for difficulties in the next update.

**Comment by William758** [comment]
Latest version should allow you to disable difficulties.
